cmd/inspr/cli/delete.go

```diff
@@ -121,7 +121,7 @@ func deleteChannels(_ context.Context, args []string) error {
 	return nil
 }
 
-func deleteCTypes(_ context.Context, args []string) error {
+func deletetypes(_ context.Context, args []string) error {
```

Comment:
I think it's better to change this func name to camel case, to maintain the project pattern